pkg/render/csi.go

```diff
@@ -423,17 +434,10 @@ func (c *csiComponent) ResolveImages(is *operatorv1.ImageSet) error {
 
 func (c *csiComponent) Objects() (objsToCreate, objsToDelete []client.Object) {
 	objs := []client.Object{c.csiDriver(), c.csiDaemonset()}
+	objs = append(objs, c.serviceAccount(), c.role(), c.roleBinding())
```

Comment:
We have actual users pushing back at us creating resources that are unused. For this reason, I think it'd be best to only render this when it is actually needed.

Reply:
Fixed.